### PR TITLE
app-admin/qtpass: Version bump to 1.1.5 and add 9999

### PR DESCRIPTION
**Pull request by SpiderX**
Gentoo-Bug: https://bugs.gentoo.org/572986
1. Bump to 1.1.5
2. EAPI 6
3. Remove dev-qt/qtsvg as not needed
4. Install man
5. Use default src_prepare (fix is in upstream)
6. Add 9999 version

Package-Manager: portage-2.3.0

### CODE REVIEW

**Comment by SpiderX** [comment]
This PL is an update for #2574 (I can't continue in #2574, so I have to start new PL) .

**Comment by a17r** [comment]
Why open a new one? You can always force-push over the old one and even edit the title for the version upgrade.

File: app-admin/qtpass/qtpass-1.1.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+RDEPEND="qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5[xcb]
+		dev-qt/qtwidgets:5
+		dev-qt/qtnetwork:5
+	)
+	!qt5? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+	)
```
Comment (SoapGentoo) [inline app-admin/qtpass/qtpass-1.1.5.ebuild]: please remove Qt4 support if possible. Qt4 is deprecated, unsupported by upstream and we'd like to remove earlier than later if possible. https://wiki.gentoo.org/wiki/Project:Qt/Policies#Handling_different_versions_of_Qt

File: app-admin/qtpass/qtpass-1.1.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+EAPI=6
+
+MY_P="QtPass-${PV}"
+
+inherit qmake-utils
+
+DESCRIPTION="multi-platform GUI for pass, the standard unix password manager"
+HOMEPAGE="https://qtpass.org/"
+SRC_URI="https://github.com/IJHack/${PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
```
Comment (SoapGentoo) [inline app-admin/qtpass/qtpass-1.1.5.ebuild]: rename to `${P}.tar.gz` instead (don't forget to regenerate manifests!)

Comment (SpiderX) [inline app-admin/qtpass/qtpass-1.1.5.ebuild]: Done

**Comment by SpiderX** [comment]
@a17r 
I have to, because I can't change branch of my fork in PL, and master doesn't have that commit. I have recently moved each ebuild in a separate branch of my fork. Sorry for any inconvenience caused.

**Comment by gktrk** [comment]
@gentoo/proxy-maint

**Comment by kensington** [comment]
Thanks a lot, picked as 0c480f4a7b76e08f34a05b06672e7b0a7d6ccaab.

**Pull request closed without merging**